test/functional/p2p_tx_privacy.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+# Copyright (c) 2022 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""
+Test that transaction announcements are only queued for peers that have
+successfully completed the version handshake.
+
+Topology:
+
+  tx_originator ----> node[0] <---- spy
+
+We test that a transaction send by tx_originator is only relayed to spy
```

Comment:
micro nit in fbcd991f8c9fe6ecdac181b09f04756cab99f47d
```suggestion
We test that a transaction sent by tx_originator is only relayed to spy
```